package.json

```diff
@@ -10,7 +10,11 @@
     "node": ">=6"
   },
   "scripts": {
-    "test": "eslint --color *.js examples/*.js && tape test.js"
+    "clean": "rm -rf coverage .nyc_output/",
+    "lint": "eslint --color *.js examples/*.js",
+    "pretest": "npm run lint",
+    "test": "tape test.js",
```

Comment:
Can you include `lint` in the test command?

Reply:
Oh, I see it is in `pretest`. Is that guaranteed to execute on `test`?

Reply:
Yes.  And is very slightly more portable.